### PR TITLE
Added ivy.array lint in docstrings

### PR DESCRIPTION
**Pull request by KareemMAX**
This PR adds [`unifyai/lint-hook`](https://github.com/unifyai/lint-hook) bot to the linting configuration, in which it scans for `ivy.array()` in the output of a docstring example and formats it

### CODE REVIEW

File: ivy/data_classes/array/array.py [hunk 1]
```diff
@@ -908,12 +908,20 @@ def __gt__(self, other):
         >>> z = x > y
         >>> print(z)
         {
-            a: ivy.array([[True, False, False],
-                          [False, False, False],
-                          [False, False, False]]),
-            b: ivy.array([[True, True, False],
-                          [False, False, False],
-                          [False, False, False]])
+            a: ivy.array(
```
Comment (VedPatwardhan) [inline ivy/data_classes/array/array.py:911]: I think the array above was originally formatted correctly, given that all frameworks format it that way. Same applies to other similar instances.

**Review by VedPatwardhan — COMMENTED** [review]
Not sure if the PR was intended to be reviewed, but just wanted to bring a case into consideration. Thanks @KareemMAX 🙂

**Review by CatB1t — CHANGES_REQUESTED** [review]
Agree to @VedPatwardhan point, also we should ideally make sure when the linter is setup, that it can handle code and string arrays differenetly, so arrays existing in code should be formatted but not the ones in the docstring.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**